src/component/registry/bpmn-elements-registry.ts

```diff
@@ -133,63 +138,98 @@ export class BpmnElementsRegistry {
    *
-   * **Note**: if you pass an id that is not related to an existing BPMN element, nothing happens on the rendering side.
+   * **Note**: If an ID is passed that does not reference an existing BPMN element, no rendering changes are made.
    *
    * @example
    * ```javascript
-   * // Remove 'highlight' from BPMN elements with id: activity_1 and activity_2
+   * // Remove 'highlight' from BPMN elements with ID: activity_1 and activity_2
    * bpmnVisualization.bpmnElementsRegistry.removeCssClasses(['activity_1', 'activity_2'], 'highlight');
    *
-   * // Remove 'running' and 'additional-info' from BPMN element with id: task_3
+   * // Remove 'running' and 'additional-info' from BPMN element with ID: task_3
    * bpmnVisualization.bpmnElementsRegistry.removeCssClasses('task_3', ['running', 'additional-info']);
    * ```
    *
-   * @param bpmnElementIds The BPMN id of the element(s) where to remove the CSS classes
-   * @param classNames The name of the class(es) to remove from the BPMN element(s)
+   * @param bpmnElementIds The BPMN ID of the element(s) from which to remove the CSS classes.
+   * @param classNames The name of the class(es) to remove from the BPMN element(s).
+   *
+   * @see {@link removeAllCssClasses} to remove all CSS classes from a BPMN element.
    */
   removeCssClasses(bpmnElementIds: string | string[], classNames: string | string[]): void {
     this.updateCssClasses(bpmnElementIds, classNames, this.cssRegistry.removeClassNames.bind(this.cssRegistry));
   }
 
   /**
-   * Toggle one/several CSS class(es) for one/several BPMN element(s).
-   * Notice that if you pass ids that are not related to existing BPMN elements, their reference will be kept within the registry but nothing happens on the rendering side.
+   * Remove any CSS classes that were previously added to one or more BPMN elements using the {@link addCssClasses} or the {@link toggleCssClasses} methods.
+   *
+   * **Note**: If an ID is passed that does not reference an existing BPMN element, no rendering changes are made.
+   *
+   * @example
+   * ```javascript
+   * // Remove all CSS classes from all BPMN elements
+   * bpmnVisualization.bpmnElementsRegistry.removeAllCssClasses();
+   *
+   * // Remove all CSS classes from BPMN elements with ID: activity_1 and activity_2
+   * bpmnVisualization.bpmnElementsRegistry.removeAllCssClasses(['activity_1', 'activity_2']);
+   *
+   * // Remove all CSS classes from BPMN element with ID: task_3
+   * bpmnVisualization.bpmnElementsRegistry.removeAllCssClasses('task_3');
+   * ```
+   *
+   * @param bpmnElementIds The BPMN ID of the element(s) from which to remove all CSS classes.
+   * If no IDs are specified, all CSS classes associated with BPMN elements will be removed.
+   *
+   * @see {@link removeCssClasses} to remove specific classes from a BPMN element.
+   * @since 0.34.0
+   */
+  removeAllCssClasses(bpmnElementIds?: string | string[]): void {
```

Comment:
**praise:** the JSDoc is correctly display in the API documentation generated by Typedoc

<!--
In the API doc preview, I only see the 1st "see" reference. I don't know if Typedoc supports several "see" tags or if they have to be put before the "param" tags.
Only the 1st "Note" is displayed 😢 
-->

![image](https://user-images.githubusercontent.com/27200110/236212014-77a74393-f147-4cf9-803a-8dfa76383c37.png)